less/variables.less

```diff
@@ -17,14 +17,14 @@
 @brand-navy-minus-10: #2e4369;
 @brand-navy: #37517e;
 
-//=== Blues (Deprecated)
+//=== Blues
 //
 //
-@brand-blue-minus-20: @brand-navy-minus-20; //#111824;
-@brand-blue-minus-10: @brand-navy-minus-20; //#192439;
-@brand-blue: @brand-navy-minus-10;          //#223049;
-@brand-blue-plus-10: @brand-navy;           //#283957;
-@brand-blue-plus-20: @brand-navy;           //#2f4366;
+@brand-blue-minus-20: #111824;
```

Comment:
Can we not use these variables in the file above instead of hardcoded colors?

Reply:
Where do we use hardcoded?  Bare in mind some of these are auto generated CSS files which cannot contain variables.

Reply:
Yeah, they always confuse me. Sorry